x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -35,6 +35,12 @@ export const ComponentStrings = {
         defaultMessage: 'Embed Object',
       }),
   },
+  ArgAddPopover: {
+    getAddAriaLabel: () =>
+      i18n.translate('xpack.canvas.argAddPopover.addAriaLabel', {
+        defaultMessage: 'Add Argument',
```

Comment:
Nit: I think all of our strings should be in sentence case, so it should be `Add argument`